mmdet/datasets/pipelines/transforms.py

```diff
@@ -75,13 +75,35 @@ def __init__(self,
 
     @staticmethod
     def random_select(img_scales):
+        """Randomly select an img_scale when ``multiscale_mode=='value'``
+
+        Args:
+            img_scales (list[tuple]): Images scales for selection.
+
+        Returns:
```

Comment:
(tuple, int): Returns a tuple ``(img_scale, scale_idx)`` where ``img_scale`` is the selected image scale and ``scale_idx`` is the selected index in the given candidates.